### PR TITLE
doc: steps clarification after going through the whole process from scratch

### PR DESCRIPTION
**Pull request by asmie**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +32,8 @@ It is possible to generate a docker image containing a working `polka-storage-no
 sudo docker build -t "pns:Dockerfile" .
 ```
 
+Please note that building may require a lot of time and resources. Linux system is preffered to build the image.
+
```
Comment (kylegranger) [inline README.md:36]: preferred

File: README.md [hunk 2]
```diff
@@ -43,3 +46,7 @@ sudo docker run pns:Dockerfile
 
 It will start the `polka-storage-node` on the local interface and two relay chain nodes. You can change the default behavior by passing your own command when running the docker image. All available options are in the [node template](https://docs.substrate.io/reference/command-line-tools/node-template/) documentation.
 
+To stop the nodes please use `docker stop` command:
+```bash
```
Comment (kylegranger) [inline README.md:50]: > `please use`

I think just `you may use` is better

File: doc/testing_guide.md [hunk 3]
```diff
@@ -76,33 +82,28 @@ yarn start
 ```
 
 ### Local parachain (collator)
-Download the `polkadot-native-storage` parachain repository:
+If you have not done this already, please download and build the polkadot-native-storage parachain repository:
 ```bash
```
Comment (kylegranger) [inline doc/testing_guide.md:86]: I think the `please` is a bit over polite here (could be deleted), but tbh, whatever you feel is best.

File: doc/testing_guide.md [hunk 4]
```diff
@@ -76,33 +82,28 @@ yarn start
...
 
 ### Putting local parachain and relay chain together
-To run a local parachain you'll need to run the local relay chain first. The following directory structure is assumed:
-- root directory
-- - polkadot-sdk
-- - polka-storage
-- - polkadot-js
-
-Ensure that `polkadot-js` is running.
+Ensure that `polkadot-js` is running, by checking browser at `localhost:3000`. For more information about Polkadot UI, please refer to the [official documentation](https://polkadot.js.org/docs/).
```
Comment (kylegranger) [inline doc/testing_guide.md]: > Ensure that

Confirm that [ more idiomatic to my ears ]

File: doc/testing_guide.md [hunk 5]
```diff
@@ -76,33 +82,28 @@ yarn start
...
 
-Run the local relay chain (two nodes):
+Run the local relay chain (two nodes) - use two different terminals, sessions (`tmux` or `screen`) or run in the background by add `&` at the end of the command:
 ```bash
```
Comment (kylegranger) [inline doc/testing_guide.md:100]: > by add

by adding

File: doc/testing_guide.md [hunk 6]
```diff
@@ -195,6 +195,12 @@ Please note that the benchmarking process is not ready for the PoC code yet and
 
 Benchmarking and updating weights should be done each time a new extrinsic is added to any pallet (weights are used to estimate transaction fees). Weights are obligatory for extrinsics that are available for users.
 
+To create a node with benchmarking capabilities you need to compile it with:
+```bash
```
Comment (kylegranger) [inline doc/testing_guide.md:199]: I think just a comma:  `capabilities, you need to compile`

File: doc/testing_guide.md [hunk 7]
```diff
@@ -195,6 +195,12 @@ Please note that the benchmarking process is not ready for the PoC code yet and
...
+cargo build --release --features runtime-benchmarks
+```
+By default, nodes using in this Testing Guide does not support benchmarking.
+
```
Comment (kylegranger) [inline doc/testing_guide.md:203]: By default, nodes used in this Testing Guide do not support benchmarking.

**Review by kylegranger — APPROVED** [review]
woo hoo!  🚢   🚀

File: doc/testing_guide.md (hunk 7 above)
Comment (asmie) [reply doc/testing_guide.md:203]: Done.

File: doc/testing_guide.md (hunk 6 above)
Comment (asmie) [reply doc/testing_guide.md:199]: Done.

File: doc/testing_guide.md (hunk 5 above)
Comment (asmie) [reply doc/testing_guide.md:100]: Done.

File: doc/testing_guide.md (hunk 4 above)
Comment (asmie) [reply doc/testing_guide.md]: Done.

File: doc/testing_guide.md (hunk 3 above)
Comment (asmie) [reply doc/testing_guide.md:86]: Done.

File: README.md (hunk 2 above)
Comment (asmie) [reply README.md:50]: Done.

File: README.md (hunk 1 above)
Comment (asmie) [reply README.md:36]: Done.

**Pull request merged**